manifest

```diff
@@ -40,16 +40,16 @@ export PACKAGES="\
 	networkmanager \
 	pipewire \
 	lib32-pipewire \
+	pipewire-alsa \
```

Comment:
Should we also install wireplumber?

Reply:
Wireplumber is installed as dependency of any of pipewire-{alsa,pulse,jack} packages.